frontend/src/component/common/StrategySeparator/StrategySeparator.tsx

```diff
@@ -1,51 +1,57 @@
 import { Box, styled, useTheme } from '@mui/material';
-import { ConditionallyRender } from '../ConditionallyRender/ConditionallyRender';
 
 interface IStrategySeparatorProps {
     text: 'AND' | 'OR';
 }
 
-const StyledContent = styled('div')(({ theme }) => ({
+const StyledAnd = styled('div')(({ theme }) => ({
     padding: theme.spacing(0.75, 1),
     color: theme.palette.text.primary,
     fontSize: theme.fontSizes.smallerBody,
     backgroundColor: theme.palette.background.elevation2,
-    borderRadius: theme.shape.borderRadius,
     position: 'absolute',
     zIndex: theme.zIndex.fab,
     top: '50%',
     left: theme.spacing(2),
     transform: 'translateY(-50%)',
     lineHeight: 1,
+    borderRadius: theme.shape.borderRadiusLarge,
+}));
+
+const StyledOr = styled(StyledAnd)(({ theme }) => ({
+    fontWeight: 'bold',
+    backgroundColor: theme.palette.background.alternative,
+    color: theme.palette.primary.contrastText,
+    left: theme.spacing(4),
 }));
 
-const StyledCenteredContent = styled(StyledContent)(({ theme }) => ({
+const StyledSeparator = styled('hr')(({ theme }) => ({
+    border: 0,
+    borderTop: `1px solid ${theme.palette.divider}`,
+    margin: 0,
+    position: 'absolute',
     top: '50%',
-    left: '50%',
-    transform: 'translate(-50%, -50%)',
-    backgroundColor: theme.palette.seen.primary,
-    borderRadius: theme.shape.borderRadiusLarge,
-    padding: theme.spacing(0.75, 1.5),
+    width: '100%',
 }));
 
 export const StrategySeparator = ({ text }: IStrategySeparatorProps) => {
     const theme = useTheme();
-
     return (
         <Box
             sx={{
                 height: theme.spacing(text === 'AND' ? 1 : 1.5),
                 position: 'relative',
-                width: '100%',
             }}
+            aria-hidden={true}
```

Comment:
not sure whether this should be hidden or not, but I've made a ticket to revisit this.